testapp/src/main/AndroidManifest.xml

```diff
@@ -20,7 +20,8 @@
 
         <activity
             android:name="com.rakuten.tech.mobile.testapp.ui.miniapplist.MiniAppListActivity"
-            android:launchMode="singleTop">
+            android:launchMode="singleTop"
+            android:exported="true">
```

Comment:
@corycaywood i am curious why MiniAppListActivity need `exported = true`, is it supposed to be accessed by outside of  Demo App?